src/serialization.cpp

```diff
@@ -132,6 +137,10 @@ static Status serializeShape(
     ov::Tensor& tensor) {
     OVMS_PROFILE_FUNCTION();
     responseOutput.clear_shape();
+    if (servableOutput->getPostProcessingHint() == TensorInfo::ProcessingHint::STRING_2D_U8) {
+        responseOutput.add_shape(tensor.get_shape()[0]);
+        return StatusCode::OK;
+    }
     auto& effectiveNetworkOutputShape = servableOutput->getShape();
     ov::Shape actualTensorShape = tensor.get_shape();
     if (effectiveNetworkOutputShape.size() != actualTensorShape.size()) {
```

Comment:
This check is important, we should not omit it in case of processing hint 2d_u8. I think this new check should be placed in line 150-151